hogwarts/scss/_article.scss

```diff
@@ -125,9 +125,41 @@ article {
 
     }
   }
-}
+
+  /* ipython notebook specific styles */
+  // Disgusting hack: We are including the minimized ipython css file
+  // which was compiled from less files as scss to make sure it only
+  // applies to the ipython_notebook parts:
+  @import "_ipython_style";
```

Comment:
only issue being font size matching the text so it might be a bit too big, but can easily be solved